src/Symfony/Component/Profiler/Profiler.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Profiler;
+
+use Symfony\Component\HttpKernel\Profiler\GenericProfileData;
```

Comment:
It might be better to move it to the Profiler, but I don't think this is a big issue as it will be removed in 3.0.